UI/EncyclopediaDetailPanel.cpp

```diff
@@ -62,6 +62,11 @@ namespace {
     const std::string PLANET_SUITABILITY_REPORT = "planet suitability report";
     const std::string GRAPH = "data graph";
     const std::string TEXT_SEARCH_RESULTS = "dynamic generated text";
+
+    /** \content_tag{CTRL_ALWAYS_REPORT} Always display a species on a planet suitability report. **/
```

Comment:
Inconsistent usage of content_tag.  We use @ as doxygen keyword indicator, not \\.